packages/core/data-transfer/src/strapi/remote/routes.ts

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line node/no-extraneous-import
+import type { Context } from 'koa';
+
+import { createTransferHandler } from './handlers';
+
+// Extend Strapi interface type to access the admin routes' API
+// TODO: Remove this when the Strapi instances will be better typed
+declare module '@strapi/strapi' {
+  interface Strapi {
+    admin: {
+      routes: {
+        method: string;
+        path: string;
+        handler: (ctx: Context) => Promise<void>;
+        config: unknown;
+      }[];
+    };
+  }
+}
+
+/**
+ * Register a transfer route in the Strapi admin router.
+ *
+ * It exposes a WS server that can be used to run and manage transfer processes.
+ *
+ * @param strapi - A Strapi instance
+ */
+export const registerAdminTransferRoute = (strapi: Strapi.Strapi) => {
+  strapi.admin.routes.push({
+    method: 'GET',
+    path: '/transfer',
```

Comment:
We'll need to append this to the url provided by the user, do we want to export the path used or hard-code it wherever we need it?

Reply:
We can export that as a constant & re-use it in the transfer commands

Reply:
After giving it some thinking, I think we ask for an exact URL in the CLI. We cannot know if the users have hidden their transfer route being some IP or custom DNS routing.

Reply:
My plan for the CLI is to require the admin path since we can't guess that (so, `example.com/admin` or `https://example.com/admin`), but not transfer, I think that route should be expected to exist just like any other admin route.

Reply:
Maybe we can switch to providing only a portion of the URL in a later step when we'll introduce this option to add a custom suffix?